tcms/testplans/urls/plans_urls.py

```diff
@@ -3,12 +3,12 @@
 from django.conf.urls import url
 
 from tcms.testplans import views
-
+from tcms.testplans.views import NewTestPlanView
 
 urlpatterns = [
     url(r'^$', views.get_all, name='plans-all'),
     url(r'^search/$', views.search, name='plans-search'),
-    url(r'^new/$', views.new, name='plans-new'),
+    url(r'^new/$', NewTestPlanView.as_view(), name='plans-new'),
```

Comment:
`views.NewTestPlanView.as_view()` so you don't have to import from the same module twice!